src/design-system/DateFormats.tsx

```diff
@@ -0,0 +1,14 @@
+import { format } from "date-fns";
+
+interface Props {
+  date: Date;
+  children?: undefined;
+}
+
+export const DateMMMD: React.FC<Props> = (props) => {
+  return <>{format(props.date, "MMM d")}</>;
+};
+
+export const DateMMMMdyyyy: React.FC<Props> = (props) => {
+  return <>{format(props.date, "MMMM d, yyyy")}</>;
+};
```

Comment:
I did a `git mv` to rename the file below to this new file, but git messes that operation up 🤷